espaloma/graphs/deploy.py

```diff
@@ -88,6 +88,17 @@ def openmm_system_from_graph(
         sys = ff.create_openmm_system(
             g.mol.to_topology(), charge_from_molecules=[g.mol]
         )
+        
```

Comment:
Can you document `charge_method` in the docstring, and maybe add something to the `README.md`?

I think we support
```python
    charge_method : str, optional, default='nn'
        Method to use for assigning partial charges:
        'nn' : Assign partial charges from the espaloma graph net model
        'am1-bcc' : Allow the OpenFF toolkit to assign AM1-BCC charges using default backend
        'gasteiger' : Assign Gasteiger partial charges (not recommended)
        'keep' : Use partial charges provided in the original `Molecule` object
```
Perhaps we could rename
* `keep` -> `from-molecule`
* `nn` -> `espaloma` to use "espaloma" charges?